### PR TITLE
Add MH3 and M3C Tokens

### PR DESCRIPTION
**Pull request by SlightlyCircuitous**
- Add Modern Horizons and Modern Horizons Commander tokens 
- Update Shapeshifter reminder text to match new printing

Given that the new Spellgorger Weird token has a mana cost of 2R and is seemingly a token copy of an existing card of the same name, should I adjust the token's cmc to 3 and put "(Token)" after the token name like with Eternalize/Embalm cards?

### CODE REVIEW

**Comment by ebbit1q** [comment]
yes, we'd like all tokens that have a similar name to a real card to have (Token) after it

File: tokens.xml [hunk 1]
```diff
@@ -1629,6 +1654,22 @@ Whenever this creature attacks, target attacking creature gains flying until end
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Cat Warrior Token</name>
```
Comment (ebbit1q) [inline tokens.xml:1658]: this is a duplicate of an existing token, it is sorted alphabetically, ie W after T

**Comment by SlightlyCircuitous** [comment]
Thanks for the fixes!

**Comment by ebbit1q** [comment]
you're welcome

**Pull request merged**